gallery/src/styles/NxCard/NxCardPageExamples.scss

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+// Reveal problems with box-sizing set to border-box, see https://issues.sonatype.org/browse/RSC-84
+
+.nx-card__content--2-column-example {
+  align-items: center;
+  display: flex;
+
+  >:first-child {
+    margin-right: 16px;
+  }
+}
+
+.nx-card--align-left {
+  text-align: left;
+}
+
+.nx-card-container--row .nx-card--75wide {
+  max-width: 75%;
+}
+
+.nx-card-container--row .nx-card--50wide {
+  max-width: 50%;
```

Comment:
Why max-width and not width for these?

Reply:
> Why max-width and not width for these?

They have to override `max-width: 330px;` set on `.nx-card`